crates/rolldown/tests/rolldown/topics/hmr/delete_file_used/artifacts.snap

```diff
@@ -0,0 +1,102 @@
+---
+source: crates/rolldown_testing/src/integration_test.rs
+---
+# Assets
+
+## main.js
+
+```js
+// HIDDEN [rolldown:runtime]
+// HIDDEN [rolldown:hmr]
+//#region child.js
+var child_exports = {};
+__export(child_exports, { value: () => value });
+const child_hot = __rolldown_runtime__.createModuleHotContext("child.js");
+__rolldown_runtime__.registerModule("child.js", { exports: child_exports });
+const value = "child";
+
+//#endregion
+//#region parent.js
+var parent_exports = {};
+__export(parent_exports, {
+	childValue: () => value,
+	parentValue: () => parentValue
+});
+const parent_hot = __rolldown_runtime__.createModuleHotContext("parent.js");
+__rolldown_runtime__.registerModule("parent.js", { exports: parent_exports });
+const parentValue = "parent";
+parent_hot.accept((newMod) => {
+	const { childValue, parentValue: parentValue$1 } = newMod;
+	assert.strictEqual(parentValue$1, "parent");
+	assert.strictEqual(childValue, "child");
+});
+
+//#endregion
+//#region main.js
+var main_exports = {};
+const main_hot = __rolldown_runtime__.createModuleHotContext("main.js");
+__rolldown_runtime__.registerModule("main.js", { exports: main_exports });
+
+//#endregion
+```
+# HMR Step 0
+## Errors
+
+### UNLOADABLE_DEPENDENCY
```

Comment:
Similar to https://github.com/rolldown/rolldown/pull/5932#discussion_r2303435006, we need to decide how to handle this.